HelpSource/scdoc.css

```diff
@@ -1,11 +1,13 @@
+
 body {
-    font-family: Arial, Helvetica, sans-serif;
+    font-family: Helvetica, Arial, sans-serif;
     font-size: 10pt;
     color: black;
     background: white;
 /*    max-width: 60em;*/
 /*    margin-left: 20em;*/
-    padding: 0;
+    width: auto;
+    padding-left: 1.5em;
```

Comment:
yes, I see. The problem is that without it, the method names are stuck to the left, abe the stars for the class methods are half-hidden behind the frame.

if you know another way, let me know.